app/coffee/AuthController.coffee

```diff
@@ -0,0 +1,57 @@
+logger = require("logger-sharelatex")
+
+AuthHandler = require("./AuthHandler")
+EmailHandler = require("./EmailHandler")
+Errors = require("./Errors")
+
+module.exports = AuthController =
+	login: (req, res, next) ->
+		{ email, password } = req.body
+		logger.info({ email }, "received password login request")
+		email = email.trim()
```

Comment:
Should we also `toLowerCase()` here? I believe that is what web does.

Reply:
Nice one! I have an item for validation in my improvements list. I'll keep this in mind.